tutorials-v5/heom/heom-1a-spin-bath-model-basic.md

```diff
@@ -507,6 +507,25 @@ plot_result_expectations(
 );
 ```
 
+The built-in class also allows us to quickly obtain the bath spectrum, correlation function, and spectral density
+
+```python
+w=np.linspace(-10,10,1000)
+w2=np.linspace(0,10,1000)
+fig, axs = plt.subplots(2, 2)
+axs[0, 0].plot(w,bath.power_spectrum(w,T))
+axs[0,0].set(xlabel=r'$\omega$',ylabel=r'$S(\omega)$')
+axs[0, 1].plot(w2,bath.spectral_density(w2))
+axs[0,1].set(xlabel=r'$\omega$',ylabel=r'$J(\omega)$')
+axs[1, 0].plot(w2,bath.CI(w2))
+axs[1,0].set(xlabel=r'$t$',ylabel=r'$C_{I}(t)$')
+axs[1, 1].plot(w2,bath.CR(w2))
+axs[1,1].set(xlabel=r'$t$',ylabel=r'$C_{R}(t)$')
+fig.tight_layout()
+plt.show()
+
+```
+
```

Comment:
This is nice to include here! But you put it a bit randomly in the middle of the discussion of the Tanimura terminator. This might fit better at the place where the DrudeLorentzBath class is introduced the first time?